content/en/docs/concepts/storage/volumes.md

```diff
@@ -543,6 +543,68 @@ spec:
       type: FileOrCreate
 ```
 
+### image
+
+{{< feature-state feature_gate_name="ImageVolume" >}}
+
+An `image` volume source represents an OCI object (a container image or
+artifact) pulled and mounted on the kubelet's host machine.
+
+One example to use the `image` volume source is:
+
+{{% code_sample file="pods/image-volumes.yaml" %}}
+
+The volume is resolved at pod startup depending on which `pullPolicy` value is
+provided:
+
+- `Always`: the kubelet always attempts to pull the reference. Container
+  creation will fail if the pull fails.
+- `Never`: the kubelet never pulls the reference and only uses a local image or
+  artifact. Container creation will fail if the reference isn't present.
+- `IfNotPresent`: the kubelet pulls if the reference isn't already present on
+  disk. Container creation will fail if the reference isn't present and the pull
+  fails.
```

Comment:
nit: I'd use a description list here.

Reply:
Do we have a special shortcode for that?